src/main/java/meteordevelopment/meteorclient/systems/modules/render/BetterTooltips.java

```diff
@@ -408,7 +408,7 @@ private void appendPreviewTooltipText(ItemStackTooltipEvent event, boolean space
         )) {
             // we don't want to add the spacer if the tooltip is hidden
             if (spacer) event.appendEnd(Text.literal(""));
-            event.appendEnd(Text.literal("Hold " + Formatting.YELLOW + keybind + Formatting.RESET + " to preview"));
+            event.appendEnd(Text.literal(String.format(Text.translatable("modules.better-tooltip.hold-for-preview").getString(), Formatting.YELLOW, keybind, Formatting.RESET)));
```

Comment:
use a single `Text.translatable()` with arguments

Reply:
![image](https://github.com/user-attachments/assets/20f68301-4a11-4b95-b315-3cc051370890)
It doesn't work with colours for some reason
![image](https://github.com/user-attachments/assets/86682d87-ebda-4236-bd3b-27cc83a96b47)

Reply:
yes, you use styled `Text` objects as arguments